src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -128,6 +160,11 @@ public void WriteWarning(string message)
             }
         }
 
+        /// <summary>
+        /// Writes a warning event log without generating an event.
```

Comment:
should be same "summary" as a above. Only difference is 'formatting parameters' are passed.